### ISSUE TITLE
Version 3.3.0 no admin email on profile update

### ISSUE THREAD

**Issue by laurelle** [MissVeronica/UM-Admin-User-Profile-Update-Email#1]
Hi,
I installed & actived the latest version of your plugin. When someone updates their profile the admin is not getting an email. When a user updates their password 2 emails are sent the the user, password reset and account update emails. But no email to the admin. 
It seems that only some fields trigger the account updated email. I need the admin to receive an email when any field in the user's profile gets updated, including custom fields. 
I have installed the Check & Log Emails and it is only showing the 2 user emails being sent out when the user updates their password. If I change a custom field in the profile, no email is sent to the user or the admin.
I have added the changedaccount_email.php and the profile_is_updated_email.php in my childs theme's /ultimate-member/templates/email folder.
Any ideas what the problem could be?
Thank you for any help.

**Comment by MissVeronica** [comment]
Hi

> I have added the changedaccount_email.php and the profile_is_updated_email.php in my childs theme's /ultimate-member/templates/email folder.

How did you install this plugin?

Install by downloading the plugin ZIP file and install as a new Plugin, which you upload in 
WordPress -> Plugins -> Add New -> Upload Plugin.
Activate the Plugin: Ultimate Member - Admin Email Profile Update

>  the profile_is_updated_email.php in my childs theme's /ultimate-member/templates/email folder.

This is correct.

**Comment by laurelle** [comment]
Hi,

Yes, I downloaded the plugin and then installed in in Wordpress using New Plugin, then Activated it.

But am not getting any Admin emails (not in Junk either) but the Check & Log email plugin is not showing that any Admin emails are being sent.

Thank you for your help

**Comment by MissVeronica** [comment]
Hi,

> When a user updates their password 2 emails are sent the the user, password reset and account update emails.

These are UM emails. If you want copies to Admin of these emails use this plugin https://github.com/MissVeronica/um-additional-email-recipients

I have tried the Profile is updated admin email and it works OK at my site.

There is an issue with UM if you have a copy of the Template email file at your Theme directory but is using a child theme.
Remove any copies of `profile_is_updated_email.php` from your theme's or child-theme's `ultimate-member/email/` folders.
Go to UM Settings -> Email and edit the Admin email template and save it, now this plugin will copy the template to the right directory.

**Comment by laurelle** [comment]
Hi,
Thank you for your help. It was the additional email recipients plugin that I really wanted and have now installed it.

I have added an email address to the "Additional Email Recipients - Bcc:" in the "Account Updated Email".
When someone updates their profile fields, I don't seem to be getting any emails to the user or to the system admin. I only get the account updated email if the user updates their password. 

Is there an email that you can get when the user updates any profile field, not just their password?
I need to know when a user has changed their profile field(s) and what they changed. Is there a way to do that? 

Thanks again for your help!

**Comment by MissVeronica** [comment]
You can look at my "UM Profile Content Moderation" plugin. 
https://github.com/MissVeronica/um-profile-content-moderation

**Comment by azmiroth** [comment]
Has anyone else had an issue with the plugin not running on PHP 8?  I get a stack error when I run on 8.

**Comment by MissVeronica** [comment]
Yes you are right, there is an issue with PHP 8 which is fixed now in the version 3.5.0

**Comment by azmiroth** [comment]
Thanks for the quick reply.  Much appreciated.  Sorry to be the bearer of bad news, but it doesn't seem to be working - although the error is no longer on line 127 - it's failing on 129 now. Uncaught TypeError: Cannot access offset of type string on string in .../public_html/wp-content/plugins/um-admin-user-profile-update-email-main/um-admin-user-profile-update-email.php:129  Running PHP v8.1.24 - also tested on 8.2.11 and 8.0.30.  Still works on 7.4.33.  Confirmed using v 3.5.0

**Comment by MissVeronica** [comment]
I can't reproduce this with PHP 8.2.9
